content/en/developers/dogstatsd/datagram_shell.md

```diff
@@ -86,11 +86,51 @@ The value is a Unix timestamp (UTC) and must be prefixed by `T`, for example:
 
 - `page.views:15|c|#env:dev|T1656581400`: A COUNT indicating that 15 page views happened on the 30th of June, 2022 at 9:30am UTC
 
+### DogStatsD protocol v1.4
+
+Starting with the Agent `>=v7.51.0`, a new inode value is supported for the container ID field.
+The container ID field can now contain two values to enrich DogStatsD metrics with additional container tags:
+- The container ID if available.
+- The cgroup node inode if the container ID is unavailable.
+
+The container ID field is still prefixed by `c:`, with the value being either:
```

Comment:
Is this accurate? Wouldn't the code still support the old non-prefixed fallback?

If the intent is to actually deprecate usage of the non-prefixed value, then we need to say there here. Otherwise, we need to indicate that it's still a viable form to use for the container ID.

Reply:
Agreed, I've added the following: https://github.com/DataDog/documentation/pull/27076/files#diff-95284c3c410e9788783a6fe2e596a0c8a530c820e2d58bd23aa3614d07d4bcb6R96